src/Instance/InstanceMarc/InstanceMarc.js

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  Pane,
+  Paneset,
+  Headline,
+} from '@folio/stripes/components';
+
+import InstanceMarcField from './InstanceMarcField';
+import {
+  isControlField,
+} from './utils';
+import styles from './InstanceMarc.css';
+
+const InstanceMarc = ({
+  instance,
+  marc,
+  onClose,
+}) => {
+  const intl = useIntl();
+
+  const renderFields = useMemo(() => ([
+    ...marc.fields.filter(isControlField),
+    ...marc.fields.filter(field => !isControlField(field)),
+  ]), [marc]);
+
+  return (
+    <Paneset isRoot>
+      <Pane
+        paneTitle={instance.title}
+        defaultWidth="fill"
+        dismissible
+        onClose={onClose}
+        data-test-instance-marc
+      >
+        <section className={styles.instanceMarcWrapper}>
+          <Headline
+            size="large"
+            margin="small"
+            tag="h3"
+          >
+            {intl.formatMessage({ id: 'ui-inventory.marcSourceRecord' })}
```

Comment:
so it's a standard now - to use formatMessage instead <FormattedMessage /> ?

Reply:
no, it's ok to use FormattedMessage - initially I had to use `intl.formatMessage` and as a result both could be used 

no difference for this case

Reply:
So I would suggest using FormattedMessage then from the general recommendation standpoint.

Reply:
what is general recommendation? :)

for me `intl.formatMessage` is better is it return just a string

Reply:
General recommendation is to use `FormattedMessage`.

Reply:
I'm ok to change to `FormattedMessage`, but could you point where it's recommended to use it?

Reply:
I remember [PRs though in the past](https://github.com/folio-org/ui-eholdings/pull/612) to use FormattedMessage when possible. Maybe @zburke or @JohnC-80 has more knowledge on this or maybe it is not actual now at all.

Reply:
I think it's because of `injectIntl`, but now we can use hooks

Reply:
Yeah, it can be true.

Reply:
There is also a [recommendation](
https://github.com/formatjs/formatjs/pull/1312/files#diff-96cc378dcdef180641176501f42183c7R98) from the react-intl devs.

> These should be used when your React component needs to format data to a string value where a React element is not suitable; e.g., a `title` or `aria` attribute, or for side-effect in `componentDidMount`.

Reply:
In `react-intl` `v2`, the `injectIntl` implementation relied on React's deprecated context implementation and this is what led to our "Prefer `<FormattedMessage>`" recommendation. With `react-intl` `v4` the `injectIntl` HOC uses the modern context API, so `intl.formatMessage()` (whether you get to `intl` via `injectIntl` or the `useIntl` hook) is just as good as `<FormattedMessage>`. The [stripes i18n documentation](https://github.com/folio-org/stripes/blob/master/doc/i18n.md) reflects this now, but I didn't do a good job publicizing this change. 

And in any case, @NikitaSedyx is correct that it was always necessary to use `formatMessage()` when you needed a string-value.

Reply:
But in this particular case, as it turned out, the string representation is not necessary, so FormattedMessage can be used instead.

Reply:
my initial comment
> no, it's ok to use FormattedMessage - initially I had to use intl.formatMessage and as a result both could be used
no difference for this case